frontend/src/app/main/ui/workspace/left_header.cljs

```diff
@@ -78,7 +79,7 @@
 
         nav-to-project
         (mf/use-fn
-         #(st/emit! (dcm/go-to-dashboard-files ::rt/new-window true)))]
+         #(st/emit! (dcm/go-to-dashboard-files ::rt/new-window true :project-id project-id)))]
```

Comment:
`project-id` should be on mf/deps of use-fn

Reply:
Thanks!